src/trafficcontroller/app/traffic_controller.go

```diff
@@ -156,23 +156,14 @@ func (t *trafficController) Start() {
 	}
 	t.startOutgoingProxy(fmt.Sprintf(":%d", t.conf.OutgoingDropsondePort), dopplerHandler)
 
-	killChan := signalmanager.RegisterKillSignalChannel()
-	dumpChan := signalmanager.RegisterGoRoutineDumpSignalChannel()
-
 	// We start the profiler last so that we can definitively claim that we're ready for
 	// connections by the time we're listening on the PPROFPort.
 	p := profiler.New(t.conf.PPROFPort)
 	go p.Start()
 
-	for {
-		select {
-		case <-dumpChan:
-			signalmanager.DumpGoRoutine()
-		case <-killChan:
-			log.Print("Shutting down")
-			return
-		}
-	}
+	killChan := signalmanager.RegisterKillSignalChannel()
```

Comment:
Again, should we just create the channel here and remove this package altogether?